package.json

```diff
@@ -35,32 +35,41 @@
   "license": "MIT",
   "dependencies": {
     "lodash.debounce": "^4.0.8",
```

Comment:
this is quite big (1kb) according to bundlephobia for such a small util, you probably only need something like this:
```js
function debounce(ms, fn) {
	let timeoutId

	return (...args) => {
		clearTimeout(timeoutId)
		timeoutId = setTimeout(fn, ms, ...args)
	}
}
```